### DISCUSSION TITLE
vela install -v v1.3.5    error:  SchemaError(dev.oam.core.v1beta1.ApplicationRevision.spec.referredObjects)

### DISCUSSION THREAD

**Discussion by skyshaokai** [kubevela/kubevela#3971] [Q&A] [answered]
minikube install vela:
vela install -v v1.3.5 

my k8s  verison : 1.19.1  

error message:
`Check Requirements ...
Installing KubeVela Core ...
Helm Chart used for KubeVela control plane installation: https://charts.kubevela.net/core/vela-core-1.3.5.tgz
Existing KubeVela installation found in namespace vela-system

Do you want to overwrite this installation? (y/n)y
I0525 00:09:27.464701   63757 apply.go:107] "patching object" name="applicationrevisions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.716875   63757 apply.go:107] "patching object" name="applications.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.775549   63757 apply.go:107] "patching object" name="componentdefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.807903   63757 apply.go:107] "patching object" name="definitionrevisions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.847871   63757 apply.go:107] "patching object" name="envbindings.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.875927   63757 apply.go:107] "patching object" name="healthscopes.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.899378   63757 apply.go:107] "patching object" name="manualscalertraits.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.924022   63757 apply.go:107] "patching object" name="policies.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.935665   63757 apply.go:107] "patching object" name="policydefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.958105   63757 apply.go:107] "patching object" name="resourcetrackers.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:27.978103   63757 apply.go:107] "patching object" name="scopedefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:28.008144   63757 apply.go:107] "patching object" name="traitdefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:28.040962   63757 apply.go:107] "patching object" name="workflows.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:28.063165   63757 apply.go:107] "patching object" name="workflowstepdefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:28.081151   63757 apply.go:107] "patching object" name="workloaddefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0525 00:09:28.112198   63757 apply.go:107] "patching object" name="rollouts.standard.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
Start upgrading Helm Chart kubevela in namespace vela-system
getting history for release kubevela
creating 1 resource(s)
CRD applicationrevisions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD applications.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD componentdefinitions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD definitionrevisions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD envbindings.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD healthscopes.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD manualscalertraits.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD policies.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD policydefinitions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD resourcetrackers.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD scopedefinitions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD traitdefinitions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD workflows.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD workflowstepdefinitions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD workloaddefinitions.core.oam.dev is already present. Skipping.
creating 1 resource(s)
CRD rollouts.standard.oam.dev is already present. Skipping.
Error: Could not install KubeVela control plane installation: error when installing/upgrading Helm Chart kubevela in namespace vela-system: unable to build kubernetes objects from release manifest: SchemaError(dev.oam.core.v1beta1.ApplicationRevision.spec.referredObjects): array should have exactly one sub-item`

**Comment by wonderflow** [comment]
What's your command line version?

**Reply by skyshaokai** [reply]
vela version
CLI Version: v1.3.4
Core Version:
GitRevision: git-d748096
GolangVersion: go1.17.9

kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T20:17:28Z", GoVersion:"go1.10.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.1", GitCommit:"206bcadf021e76c27513500ca24182692aabd17e", GitTreeState:"clean", BuildDate:"2020-09-09T11:18:22Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}

**Comment by Somefive** [comment, marked as answer]
Try solutions in https://github.com/kubevela/kubevela/issues/3874 and see if it can solve you problem.
